exercises/concept/ticket-please/.docs/hints.md

```diff
@@ -0,0 +1,47 @@
+# Hints
+
+## General
+
+- Read about [pattern matching][pattern-matching] in the official guide.
+- Have a look at this cheat-sheet about [destructuring][destructuring].
+- Pattern matching for records is [explained here][records-pattern-matching] in more details.
+
+## 1. Detect empty comments
+
+- You can use `(a, b)` directly in the function argument if the argument is a pair.
+- You can replace values that you receive but don't use by the wildcard `_`.
+- To check if a string is empty, check the [`String` documentation][string-docs].
+
+## 2. Count the number of comments from the creator
+
+- You can destructure a custom type with a single variant `type Box = Box AB` directly in a function argument like so: `(Box ab)`.
+- You can pattern match the field `a` of a record `{a : A, b : B}` using: `{ a }`.
+- You can use both the above techniques at the same time for custom types that hold a record.
+- You cannot directly pattern match the pair `ab` from a record `{ ab : (A, B) }`. You must pattern match `{ ab }` first, then use `( a, b) = ab` in a `let` binding.
+- To find the number of elements that satisfy a condition in a list, use [`List` functions][list-docs].
+- You can destructure pairs diectly in anonymous functions arguments.
+
+## 3. Detect tickets assigned to the Dev Team
+
+- You can destructure a custom type with a single variant `type Box = Box AB` directly in a function argument like so: `(Box ab)`.
```

Comment:
A lot of these hints are repeated in the various sections, maybe they could go in the General section?